tests/test_tags.py

```diff
@@ -44,13 +42,7 @@ def manylinux_module(monkeypatch):
 @pytest.fixture
 def mock_interpreter_name(monkeypatch):
     def mock(name):
-        if hasattr(sys, "implementation") and sys.implementation.name != name.lower():
-            monkeypatch.setattr(sys.implementation, "name", name.lower())
-            return True
-        elif platform.python_implementation() != name:
-            monkeypatch.setattr(platform, "python_implementation", lambda: name)
-            return True
-        return False
+        monkeypatch.setattr(sys.implementation, "name", name.lower())
```

Comment:
This actually is too aggressive. The conditional check as to whether to do the monkeypatching was necessary was on purpose to avoid running against monkeypatched code _all_ the time.
```suggestion
        name = name.lower()
        if sys.implementation.name != name:
            monkeypatch.setattr(sys.implementation, "name", name)
            return True
        return False
```

This also means all the other tests where the `if` check on the return value of this function was removed should be reverted.

Reply:
Thanks for the reveiw! I see what you're saying. I've applied the suggestion in the latest revision and commit "Partially revert changes to mock_interpreter_name()".